lib/analyzer.js

```diff
@@ -204,6 +205,9 @@ Analyzer.analyze = function analyze(href, options) {
                         require('./loader/fs-resolver') :
                         require('./loader/xhr-resolver');
   loader.addResolver(new PrimaryResolver(options));
+  if (options.content) {
```

Comment:
It would be helpful to document `options.content` on the annotation for LoadOptions above